src/wp-includes/class-wp-image-editor.php

```diff
@@ -213,6 +214,28 @@ protected function update_size( $width = null, $height = null ) {
 		return true;
 	}
 
+	/**
+	 * Gets the current image size name.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return string Image size name, or empty string if none set.
+	 */
+	public function get_size_name() {
+		return $this->size_name;
+	}
+
+	/**
+	 * Sets the current image size name.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param string $size_name The image size name.
+	 */
+	protected function update_size_name( $size_name ) {
```

Comment:
I would expect this to be named "set_size_name", but I guess this matches other functions in this file?

Reply:
@adamsilverstein Correct, that's why I chose that name. I don't feel strongly about it though, LMK what you prefer.

Reply:
i defer to you in all naming decisions :) i agree it makes sense to keep the pattern established in the file already, it just stands out in the PR because I'm so used to the `set_` nomenclature.